### PR TITLE
fix(Spawning/Pathing) An'daroth, Ghostlands

### PR DESCRIPTION
**Pull request by malcrom**
## Changes Proposed:
-  Respawn, Add pathing, script Sentinel Leader

## SOURCE:
Sniff

## Tests Performed:
- compiles and inserts
- Tested in game

## How to Test the Changes:
1. .g c 82157
2. .g c 82139
3. .g c 82136

### CODE REVIEW

**Review by IntelligentQuantum — APPROVED** [review]

File: src/server/scripts/EasternKingdoms/zone_ghostlands.cpp [hunk 1]
```diff
@@ -143,7 +143,137 @@ class npc_ranger_lilatha : public CreatureScript
...
+    SPELL_SINISTER_STRIKE = 14873,
+    SPELL_BACKSTAB        = 7159
+};
+
+class npc_sentinel_leader : public CreatureScript
+{
+public:
+    npc_sentinel_leader() : CreatureScript("npc_sentinel_leader") {}
+
+    struct npc_sentinel_leaderAI : public ScriptedAI
```
Comment (Kitzunu) [inline src/server/scripts/EasternKingdoms/zone_ghostlands.cpp:166]: Use new script model

**Review by Kitzunu — CHANGES_REQUESTED** [review]

File: src/server/scripts/EasternKingdoms/zone_ghostlands.cpp (hunk 1 above)
Comment (Kitzunu) [reply src/server/scripts/EasternKingdoms/zone_ghostlands.cpp:166]: RegisterCreatureAI

Comment (malcrom) [reply src/server/scripts/EasternKingdoms/zone_ghostlands.cpp]: done

**Review by Kitzunu — APPROVED** [review]

**Comment by temperrr** [comment]
Tested


- [x] Walks around, talks to other npcs
- [x] uses sinister strike
- [x] uses backstab when back is turned

**Comment by FrancescoBorzi** [comment]
when content-specific PR (e.g. 1-19) aren't linked to any open issue, like in this case, please add the Project to the PR as well

**Pull request merged**